src/Components/Profile/MyPosts/myPosts.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Post from './Post/post'
+
+import s from './myPosts.module.css';
+
+const MyPosts = () => {
+  return (
+    <div>
+      My posts
+        <div></div>
+      <textarea></textarea>
+      <button>Add post</button>
+      <div className={s.posts}>
+        <Post message='Hi,How are you?' />
```

Comment:
Consider storing such strings in a separate "constants" file on the root level of your app.
E.g. create a folder "core" under "src" and store the constants file there.
Applies even for temporary strings.

Reply:
well, I will try to do it before the next commit